scripts/libs/libraries.ts

```diff
@@ -177,6 +178,16 @@ export async function deployLibraries(owner: Signer): Promise<LibraryAddresses>
                 Key32Lib: key32LibAddress,
             }
         });
+
```

Comment:
when you add a new library here, please ensure that its address is also printed in the `printAddresses` function in the `deploy_all.ts` script.

Reply:
done